src/Data/Profunctor/Traversing.hs

```diff
@@ -42,6 +46,15 @@ secondTraversing = traverse'
 swapE :: Either a b -> Either b a
 swapE = either Right Left
 
+dimapTraversing :: Traversing p => (a' -> a) -> (b -> b') -> p a b -> p a' b'
+dimapTraversing f g = wander (\afb a' -> g <$> afb (f a'))
+
+lmapTraversing :: Traversing p => (a -> b) -> p b c -> p a c
+lmapTraversing f = wander (\afb a' -> afb (f a'))
```

Comment:
`lmapTraversing` may be more efficient than the default definition. `rmapTraversing` won't be, but it seems asymmetrical to have one and not the other.